include/psa/crypto_sizes.h

```diff
@@ -573,10 +585,12 @@
 /** A sufficient output buffer size for psa_asymmetric_encrypt(), for any
  *  supported asymmetric encryption.
  *
+ * This macro assumes that RSA is the only supported asymmetric encryption.
```

Comment:
I don't think this should be part of the doxygen documentation. The macro should always be correct even if we add other types in the future (though that seems unlikely). Please make this a non-doxygen comment (just like you did for FFDH): "We only support RSA for asymmetric encryption." (Same below.)